KarhooUISDK/Screens/PrebookConfirmationScreen/KarhooBookingConfirmationViewModel.swift

```diff
@@ -174,6 +176,14 @@ class KarhooBookingConfirmationViewModel: BookingConfirmationViewModel {
         return dateFormatter.display(clockTime: journeyDetails.scheduledDate)
     }
 
+    var accessibilityDate: String {
+        dateFormatter.display(fullDate: journeyDetails.scheduledDate)
+    }
+
+    var accessibilityPrice: String {
+        quote.quoteType.description + ", " + CurrencyCodeConverter.toPriceString(quote: quote)
```

Comment:
This description will be incorrect (in english version) for mettered price, because description for metered is "Metered" (for another cases it is "Estimated price" and "Fixed price"). We need to change description on create another value for quoteType different than description and add text "price" to make accessibility phrase more descriptive